front-end/src/App.tsx

```diff
@@ -44,20 +43,23 @@ function App() {
         <div>
           <Container>
             <Row style={{padding: '3rem'}} className="justify-content-md-center">
-              <Col xs="2"><AppNavBar loggedInUser={loggedInUser} setLoggedInUser={setLoggedInUser} /></Col>
+              <Col md="2">
+                <AppNavBar loggedInUser={loggedInUser} setLoggedInUser={setLoggedInUser} /></Col>
               <Col>
               <Switch>
               <Route exact path="/" render={(props) => <HomePage {...props} loggedInUser={loggedInUser} />} />
-              <Route path="/auth" render={(props) => <AuthPage {...props} setLoggedInUser={setLoggedInUser} />} />
+              <Route path="/login" render={(props) => <AuthPage {...props} setLoggedInUser={setLoggedInUser} />} />
+              {loggedInUser && 
```

Comment:
I'm not sure if this could cause bugs for some features (links and stuff).
Here's how to do React Router Auth redirects, though it may take more time than what we have left. https://reactrouter.com/web/example/auth-workflow
Personally, I'd rather go without this change since our current workflow and demo work is alright

Reply:
could u point to me what change I should revert? Honestly if any of us doesn't like something or is unsure of any potential issues I think we should just scrape it

Reply:
After thinking about it and playing around with it a bit your implementation looks good for our demo :rocket: . If we find anything we can catch it during our test run tomorrow